ocs_sample_library_preview/OCSClient.py

```diff
@@ -1,92 +1,101 @@
+from .Assets import Assets
 from .BaseClient import BaseClient
 from .DataViews import DataViews
-from .Types import Types
 from .Streams import Streams
+from .Types import Types
 
 
 class OCSClient:
     """
     A client that handles communication with OCS
     """
 
-    def __init__(self, apiversion, tenant, url, clientId, clientSecret=None,
-                 acceptVerbosity=False):
+    def __init__(self, api_version: str, tenant: str, url: str, client_id: str,
+                 client_secret: str = None, accept_cerbosity: bool = False):
         """
-        Use this to help communinication with OCS
-        :param apiversion: Version of the api you are communicating with
+        Use this to help in communinication with OCS
+        :param api_version: Version of the api you are communicating with
         :param tenant: Your tenant ID
         :param url: The base URL for your OCS instance
-        :param clientId: Your client ID
-        :param clientSecret: Your client Secret or Key
-        :param acceptVerbosity: Sets whether in value calls you get all values
-                or just non-default values
-        """
-        self.__baseClient = BaseClient(apiversion, tenant, url, clientId,
-                                       clientSecret, acceptVerbosity)
-        self.__DataViews = DataViews(self.__baseClient)
-        self.__Types = Types(self.__baseClient)
-        self.__Streams = Streams(self.__baseClient)
+        :param client_id: Your client ID
+        :param client_secret: Your client Secret or Key
+        :param accept_verbosity: Sets whether in value calls you get all values or just
+            non-default values
+        """
+        self.__base_client = BaseClient(api_version, tenant, url, client_id,
+                                        client_secret, accept_cerbosity)
+        self.__assets = Assets(self.__base_client)
+        self.__data_views = DataViews(self.__base_client)
+        self.__streams = Streams(self.__base_client)
+        self.__types = Types(self.__base_client)
 
     @property
-    def uri(self):
+    def uri(self) -> str:
         """
         :return: The uri of this OCS client as a string
         """
-        return self.__baseClient.uri
+        return self.__base_client.uri
 
     @property
-    def tenant(self):
+    def tenant(self) -> str:
         """
         :return: The tenant of this OCS client as a string
         """
-        return self.__baseClient.tenant
+        return self.__base_client.tenant
 
     @property
-    def acceptverbosity(self):
+    def acceptverbosity(self) -> bool:
```

Comment:
should the property names in this class be CapitalWithNoSpaces to be consistent with the other public getters and setters?

Reply:
Potentially, but I wanted to avoid adding too many changes that will break existing samples, and that would likely break anything that's using this package.

Reply:
right, ignore me. that's a very impactful change that we wouldn't want to do.